src/redux/chromeReducers.ts

```diff
@@ -338,7 +339,56 @@ export function toggleNotificationsReducer(state: ChromeState) {
     ...state,
     notifications: {
       ...state.notifications,
+      data: testData,
       isExpanded: !state.notifications?.isExpanded,
     },
   };
 }
+
+export function markNotificationAsRead(state: ChromeState, { payload }: { payload: number }): ChromeState {
+  return {
+    ...state,
+    notifications: {
+      isExpanded: state.notifications?.isExpanded || false,
```

Comment:
Follow-up work for different tasks. The `state.notifications` should not be optional. We can instantiate it with empty array.

Reply:
Hey so those are mostly for the compiler/linter not to complaint, though I did just add the initial state for the notifications as well based on this.